### PR TITLE
Improve the AutoMM beginner tutorial

### PR DESCRIPTION
**Pull request by zhiqiangdon**
1. Create a tutorial dataset (by binarizing Petfinder's labels and subsampling the dataset) used in the multimodal tutorial for beginners.
2. Change metric from `accuracy` to `roc_auc`.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1908-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1908/1/index.html

File: docs/tutorials/multimodal/beginner_multimodal.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ np.random.seed(123)
 
 ## Dataset
 
-For demonstration, we use the [PetFinder dataset](https://www.kaggle.com/c/petfinder-adoption-prediction). The PetFinder dataset provides information about shelter animals that appear on their adoption profile to predict the animals' adoption rates, grouped into five categories, hence a multi-class classification problem.
+For demonstration, we use a simplified and subsampled version of [PetFinder dataset](https://www.kaggle.com/c/petfinder-adoption-prediction), which provides information about shelter animals that appear on their adoption profile to predict the animals' adoption rates, grouped into two categories, hence a binary classification problem.
```
Comment (bryanyzhu) [inline docs/tutorials/multimodal/beginner_multimodal.md:18]: Can you add one more sentence here to clarify what are the two categories? That should be helpful for beginners.

**Review by bryanyzhu — APPROVED** [review]
LGTM, except one minor comment, because I'm not sure about the meaning of `to_be_adopted`

**Comment by szha** [comment]
Job PR-1908-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1908/3/index.html

**Review by sxjscience — APPROVED** [review]
LGTM. Thanks!!

**Comment by szha** [comment]
Job PR-1908-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1908/4/index.html

**Pull request merged**